### DISCUSSION TITLE
Copy config from one computer/profile to a new one

### DISCUSSION THREAD

**Discussion by reuvygroovy** [Authenticator-Extension/Authenticator#951] [Q&A]
I migrated to a new profile/computer. Is there a way to copy/import/export the data from one PC to another PC?

**Comment by IgnoreBox** [comment]
i just test it
u only need to download/export config
and then upload/import

**Comment by reuvygroovy** [comment]
Where is the config stored?

**Reply by IgnoreBox** [reply]
i found the answer,,, i never know the feature already exist

https://github.com/Authenticator-Extension/Authenticator/discussions/957#discussioncomment-3753549

**Comment by IgnoreBox** [comment]
depend on where you export it
just go to backup then choose backup type...
copy the json file 
then on target browser, just simple import

**Comment by reuvygroovy** [comment]
Where is there a backup setting? Where is the json file?
